test/sycl_tests.cc

```diff
@@ -148,4 +148,21 @@ TEST_CASE_METHOD(test_utils::device_queue_fixture, "SYCL can access empty buffer
 	}
 }
 
+#if CELERITY_FEATURE_SIMPLE_SCALAR_REDUCTIONS
+
+TEST_CASE_METHOD(test_utils::device_queue_fixture, "SYCL has working simple scalar reductions", "[sycl][reductions]") {
```

Comment:
Does this still fail anywhere? If so, maybe add a comment saying that if this test fails, Celerity can't reliably support reductions on the user's combination of backend and hardware.

Reply:
Yes, it does still fail. I added the comment.